.env.example

```
@@ -1,6 +1,6 @@
-APP_NAME=Laravel
+APP_NAME=ToDos
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:/485v/5Y9pzzH1dqseQDm1pg40qJmvh753F3eFtf/g0=
```

APP_KEYは公開しない情報になります
おそらく `cp .env .env.example` とされたのかと思いますが、

一度バージョン管理に上げてしまうと行を削除する差分を追加するだけではダメで、ローカル／リモートの履歴から削除、バージョン管理のサービスに連絡してキャッシュからも消してもらう等の対応をすることになります。

今回は運用していないトイアプリケーションなので、 .env.exampleは

```suggestion
APP_KEY=
```

とした上で、APP_KEYを生成しなおしてください :pray: